typify-impl/src/lib.rs

```diff
@@ -554,7 +588,7 @@ impl TypeSpace {
                 )
             }
         };
-        let entry_name = type_entry.name().unwrap().clone();
+        let entry_name = type_entry.name().expect("Get name for type entry").clone();
```

Comment:
I'm not sure I see how this error message is an improvement or for whom.

Reply:
It's easier to identify the line number if you're looking at source code that's not exactly the same commit as that of the code you're executing. But the text itself is not very helpful, I admit. As I said, this PR is quick and dirty and full of guesswork. I'd think of it as a sketch.

Reply:
Also error messages are nice. Memorising the line numbers of unwraps and their meanings is clearly less nice.